contact/views.py

```diff
@@ -28,6 +42,12 @@ def contact(request, topic=None, erp_slug=None):
             )
             message.sent_ok = sent_ok
             message.save()
+
+            send_receipt(
+                message.email,
+                message.get_topic_display(),
+                message.topic == "vaccination",
```

Comment:
We should probably want to use `message.topic == Message.TOPIC_VACCINATION` here